polaris.shopify.com/README.md

```diff
@@ -16,3 +16,75 @@ To quickly create new components, run:
 ```
 yarn create-component
 ```
+
+## Site Content
+
+Our website combines next.js pages with [MDX](https://mdxjs.com/) markdown files. Here's a guide to understanding the content directory structures and how they translate to pages within the website.
+
+### `/content`
+
+The root `content` directory contains markdown files for all website pages. The structure within `/content` directly corresponds to the site's navigation menu and URL pattern.
+
+For instance, `/content/design/index.md` corresponds to `polaris.shopify.com/design`, while `/content/design/colors.md` corresponds to `polaris.shopify.com/design/colors`.
+
+#### Adding New Pages
+
+To add a new page to `polaris.shopify.com`, insert a markdown `.md` file into the `/content` directory.
+
+For instance, to create `polaris.shopify.com/awesome-new-page`, add `awesome-new-page.md` to the `/content` directory. The directory structure would then be:
+
+```
+├── content
+│   ├── awesome-new-page.md
+```
+
+#### Adding New Section Pages
+
+To add a new page to an existing section on `polaris.shopify.com`, add a markdown `.md` file to the appropriate directory.
+
+For instance, to create `polaris.shopify.com/design/awesome-new-page`, add `awesome-new-page.md` to the `/content/design` directory. The directory structure would then be:
+
+```
+├── content
+│   ├── design
+│   │   ├── awesome-new-page.md
+```
+
+#### Adding a New Site Section
+
+To create a new section with a landing page at `polaris.shopify.com/your-new-page`, create a new directory within `/content` and add an `index.md` file for the landing page content. The structure would then be:
+
+```
+├── content
+│   ├── your-new-section
+│   │   ├── index.md
+```
+
+To add more pages to your new section, add more markdown `.md` files to the same directory.
+
+```
+├── content
+│   ├── your-new-section
+│   │   ├── index.md
+│   │   ├── awesome-new-page.md
+```
+
+`/content/your-new-section/your-new-page.md` will now correspond to `polaris.shopify.com/your-new-section/awesome-new-page`.
```

Comment:
```suggestion
`/content/your-new-section/awesome-new-page.md` will now correspond to `polaris.shopify.com/your-new-section/awesome-new-page`.
```

Not sure if we want to match the directory example or line 72, but one says `awesome-new-page.md` and one says `your-new-page.md`